hackathon_site/dashboard/frontend/src/components/cart/CartCard/CartCard.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import styles from "./CartCard.module.scss";
+import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+import {
+    Card,
+    CardMedia,
+    Select,
+    FormControl,
+    MenuItem,
+    IconButton,
+    Paper,
+    InputLabel,
+} from "@material-ui/core";
+import CloseIcon from "@material-ui/icons/Close";
+
+const Selections = (currentStock) => {
+    const items = [];
+    let i = 0;
+
+    for (i = 1; i < currentStock + 1; i++) {
+        items.push(
+            <MenuItem key={i} role="quantity" value={i.toString()}>
+                {i}
+            </MenuItem>
+        );
+    }
+    return items;
+};
+
+export const CartCard = ({
+    image,
+    title,
+    currentStock,
+    closeChange,
+    handleChange,
+    item,
```

Comment:
Another prop you need is the quantity of the object they checked out. The dropdown should be displaying that number

Reply:
For now, you should move 

```javascript
const [item, setItem] = React.useState(checkedOutQuantity);
const handleChange = (event) => {
        setItem(event.target.value);
};
```
inside CartCard.js, and `checkedOutQuantity` should be an integer prop you pass in that is the quantity that they checked out